### ISSUE TITLE
Add: sikh, sikhs, sikhism

### ISSUE THREAD

**Issue by MrPie5** [freebee-game/enable#116]

**Comment by PickledThyme** [comment]
those are proper nouns though...

**Comment by MrPie5** [comment]
Yeah, coming back to this maybe they shouldn't be added. When I posted this I looked at the dictionary and saw catholic and I thought christian (though I don't see that now so maybe it was removed or I misread), so I figured religions were fair game. Looks like catholic actually has a non-proper version though.
